test/unit/utility.test.js

```diff
@@ -0,0 +1,21 @@
+const { convertToPence, convertToPounds, reverseInvoiceNumber } = require('../../app/utility')

```

Comment:
Shouldn't this be moved to under its own `utility` folder?

Also, should we not have each of these files tested separately like we have throughout the services?

Reply:
Can see you have moved each func into it's own file -- can you split this test to match those files?